src/legacy/core_plugins/kibana/server/tutorials/apm/instructions/apm_agent_instructions.js

```diff
@@ -304,52 +304,66 @@ export const createRackAgentInstructions = (apmServerUrl = '', secretToken = '')
-      defaultMessage: 'Please refer to [the documentation]({documentationLink}).',
-      values: { documentationLink: '{config.docs.base_url}guide/en/apm/server/{config.docs.version}/rum.html' },
+      defaultMessage: 'APM Server disables RUM support by default. See the [documentation]({documentationLink}) \
+for details on how to enable RUM support.',
+      values: { documentationLink: '{config.docs.base_url}guide/en/apm/server/{config.docs.version}/configuration-rum.html' },
     }),
   },
   {
-    title: i18n.translate('kbn.server.tutorials.apm.jsClient.install.title', {
-      defaultMessage: 'Install the APM agent',
-    }),
-    textPre: i18n.translate('kbn.server.tutorials.apm.jsClient.install.textPre', {
-      defaultMessage: 'Install the APM agent for JavaScript as a dependency to your application:',
-    }),
-    commands: [`npm install @elastic/apm-rum --save`],
-  },
-  {
-    title: i18n.translate('kbn.server.tutorials.apm.jsClient.configure.title', {
-      defaultMessage: 'Configure the agent',
+    title: i18n.translate('kbn.server.tutorials.apm.jsClient.installDependency.title', {
+      defaultMessage: 'Set up the Agent as a dependency',
     }),
-    textPre: i18n.translate('kbn.server.tutorials.apm.jsClient.configure.textPre', {
-      defaultMessage: 'Agents are libraries that run inside of your application.',
+    textPre: i18n.translate('kbn.server.tutorials.apm.jsClient.installDependency.textPre', {
+      defaultMessage: 'You can install the Agent as a dependency to your application with \
+`npm install @elastic/apm-rum --save`.\n\n\
+The Agent can then be initialized and configured in your application like this:',
     }),
     commands: `import {curlyOpen} init as initApm {curlyClose} from '@elastic/apm-rum'
 var apm = initApm({curlyOpen}
 
-  // ${i18n.translate('kbn.server.tutorials.apm.jsClient.configure.commands.setRequiredServiceNameComment', {
+  // ${i18n.translate('kbn.server.tutorials.apm.jsClient.installDependency.commands.setRequiredServiceNameComment', {
     defaultMessage: 'Set required service name (allowed characters: a-z, A-Z, 0-9, -, _, and space)',
   })}
   serviceName: '',
 
-  // ${i18n.translate('kbn.server.tutorials.apm.jsClient.configure.commands.setCustomApmServerUrlComment', {
+  // ${i18n.translate('kbn.server.tutorials.apm.jsClient.installDependency.commands.setCustomApmServerUrlComment', {
     defaultMessage: 'Set custom APM Server URL (default: {defaultApmServerUrl})',
     values: { defaultApmServerUrl: 'http://localhost:8200' },
   })}
   serverUrl: '${apmServerUrl}',
 
-  // ${i18n.translate('kbn.server.tutorials.apm.jsClient.configure.commands.setServiceVersionComment', {
-    defaultMessage: 'Set service version (required for sourcemap feature)',
+  // ${i18n.translate('kbn.server.tutorials.apm.jsClient.installDependency.commands.setServiceVersionComment', {
+    defaultMessage: 'Set service version (required for source map feature)',
```

Comment:
Is there any way to make the code highlighting not apply to the `for` and `source` words in this comment?

Reply:
The highlighting is wrong. It looks like it's not using the javascript highlighter. I'm not sure this is something you can change from the tutorial. 

Afair these tutorials will be replaced by something else. Do you know if that's correct? If so I'm not sure it makes sense to spend too much time on. It's currently a bit of a mess and I don't think anyone owns this area, so would take some effort from us to fix.